packages/experimental/Switch/src/Switch.types.ts

```diff
@@ -126,6 +126,14 @@ export interface SwitchTokens extends LayoutTokens, FontTokens, IBorderTokens, I
   above?: SwitchTokens;
 }
 
+export interface IAnimationConfig {
```

Comment:
Interface names don't need to start with `I` anymore

Reply:
Removed